docs/config.json

```diff
@@ -86,275 +86,289 @@
       "moved": 0
     },
     {
-      "quadrant": 0, 
```

Comment:
I'm guessing something like `prettier` did all this formatting? Nice, ideally we should put this in as a PR check too but it's fiiine :D

Reply:
Ahhh I think it's just my vscode setting which is trimming whitespaces at the end of lines, sorry about that! 🙈